### ISSUE TITLE
<font size="x">

### ISSUE THREAD

**Issue by dmanthing** [wbb/WysiBB#77]
The size attribute of <font> is not supported in HTML5. Use CSS instead.
How can I do a switch over to css instead of parsing as font size="x"

**Comment by HDVinnie** [comment]
yes I have same issue....font sizes don't work. Is this project really dead?
